pkg/systemstatsmonitor/disk_collector.go

```diff
@@ -290,7 +303,10 @@ func (dc *diskCollector) collect() {
 		dc.mBytesUsed.Record(map[string]string{deviceNameLabel: deviceName, fsTypeLabel: fstype, mountOptionLabel: opttypes, stateLabel: "free"}, int64(usageStat.Free))
 		dc.mBytesUsed.Record(map[string]string{deviceNameLabel: deviceName, fsTypeLabel: fstype, mountOptionLabel: opttypes, stateLabel: "used"}, int64(usageStat.Used))
 	}
-
+	if dc.mBootDiskSize == nil {
```

Comment:
(nit) suggest to write it like:

if dc.mBootDiskSize != nil {
 dc.recordBootDiskSize(dc.config.LsblkTimeout)
}

Then it will not prevent you from adding more metrics in future.